osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -386,9 +386,8 @@ private void joinedChannelsChanged(object? sender, NotifyCollectionChangedEventA
                     {
                         channelList.RemoveChannel(channel);
 
-                        if (loadedChannels.ContainsKey(channel))
+                        if (loadedChannels.TryGetValue(channel, out var loaded))
```

Comment:
Code inspector suggests changing this to:
```csharp
                        if (loadedChannels.Remove(channel, out var loaded))
                        {
                            // DrawableChannel removed from cache must be manually disposed
                            loaded.Dispose();
                        }
```
Which sounds good.

Reply:
There are CA1853 and CA1854 for interaction with getter and Remove, but no rule for them three. Looks fine.